<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite errors="0" failures="2" hostname="DESKTOP-QCTIMVE" name="pytest" skipped="0" tests="8" time="0.256" timestamp="2022-10-16T17:19:19.675233"><testcase classname="tests.test_calc" file="tests/test_calc.py" line="3" name="test_add_01" time="0.005"></testcase><testcase classname="tests.test_calc" file="tests/test_calc.py" line="7" name="test_add_02" time="0.001"></testcase><testcase classname="tests.test_calc" file="tests/test_calc.py" line="11" name="test_dif_01" time="0.001"></testcase><testcase classname="tests.test_calc" file="tests/test_calc.py" line="15" name="test_dif_02" time="0.001"></testcase><testcase classname="tests.test_calc" file="tests/test_calc.py" line="19" name="test_seki_01" time="0.001"></testcase><testcase classname="tests.test_calc" file="tests/test_calc.py" line="23" name="test_seki_02" time="0.002"></testcase><testcase classname="tests.test_calc" file="tests/test_calc.py" line="27" name="test_shou_01" time="0.002"><failure message="assert 4 == 4.5
 +  where 4 = &lt;bound method Calc.shou of &lt;main.calc.Calc instance at 0x7f50b5e38fa0&gt;&gt;()
 +    where &lt;bound method Calc.shou of &lt;main.calc.Calc instance at 0x7f50b5e38fa0&gt;&gt; = &lt;main.calc.Calc instance at 0x7f50b5e38fa0&gt;.shou
 +      where &lt;main.calc.Calc instance at 0x7f50b5e38fa0&gt; = Calc(9, 2)">def test_shou_01():
&gt;       assert Calc(9, 2).shou() == 4.5
E       assert 4 == 4.5
E        +  where 4 = &lt;bound method Calc.shou of &lt;main.calc.Calc instance at 0x7f50b5e38fa0&gt;&gt;()
E        +    where &lt;bound method Calc.shou of &lt;main.calc.Calc instance at 0x7f50b5e38fa0&gt;&gt; = &lt;main.calc.Calc instance at 0x7f50b5e38fa0&gt;.shou
E        +      where &lt;main.calc.Calc instance at 0x7f50b5e38fa0&gt; = Calc(9, 2)

tests/test_calc.py:29: AssertionError</failure></testcase><testcase classname="tests.test_calc" file="tests/test_calc.py" line="31" name="test_shou_02" time="0.002"><failure message="assert -5 == -4.5
 +  where -5 = &lt;bound method Calc.shou of &lt;main.calc.Calc instance at 0x7f50b5ec8500&gt;&gt;()
 +    where &lt;bound method Calc.shou of &lt;main.calc.Calc instance at 0x7f50b5ec8500&gt;&gt; = &lt;main.calc.Calc instance at 0x7f50b5ec8500&gt;.shou
 +      where &lt;main.calc.Calc instance at 0x7f50b5ec8500&gt; = Calc(-9, 2)">def test_shou_02():
&gt;       assert Calc(-9, 2).shou() == -4.5
E       assert -5 == -4.5
E        +  where -5 = &lt;bound method Calc.shou of &lt;main.calc.Calc instance at 0x7f50b5ec8500&gt;&gt;()
E        +    where &lt;bound method Calc.shou of &lt;main.calc.Calc instance at 0x7f50b5ec8500&gt;&gt; = &lt;main.calc.Calc instance at 0x7f50b5ec8500&gt;.shou
E        +      where &lt;main.calc.Calc instance at 0x7f50b5ec8500&gt; = Calc(-9, 2)

tests/test_calc.py:33: AssertionError</failure></testcase></testsuite></testsuites>